pkg/agent/agent.go

```diff
@@ -454,11 +485,21 @@ func (i *Initializer) initNodeLocalConfig() error {
 		return err
 	}
 
-	i.nodeConfig = &types.NodeConfig{Name: nodeName, PodCIDR: localSubnet}
+	localAddr, dev, err := util.GetDefaultLocalNodeAddr()
+	if err != nil {
+		return fmt.Errorf("initNodeLocalConfig %v", err)
+	}
+
+	servRtTable := &types.ServiceRtTableConfig{Idx: 254, Name: "main"}
```

Comment:
Could we move line 488-497 to package route, as it needs specific knowledge like Idx/Name?

Reply:
done.